wandb/sdk/internal/incremental_table_util.py

```diff
@@ -0,0 +1,46 @@
+import time
+from typing import TYPE_CHECKING
+
+import wandb
+
+if TYPE_CHECKING:
+    from wandb import Table
+    from wandb.sdk.artifacts.artifact import Artifact
+    from ..wandb_run import Run as LocalRun
+
+ART_TYPE = "wandb-run-incremental-table"
+
+
+def init_artifact(run: "LocalRun", sanitized_key: str) -> "Artifact":
+    """Initialize a new artifact for an incremental table.
+    
+    Args:
+        run: The wandb run associated with this artifact
+        sanitized_key: Sanitized string key to identify the table
+    
+    Returns:
+        A wandb Artifact configured for incremental table storage
+    """
+    artifact_name = f"run-{run.id}-incr-{sanitized_key}"
+    artifact = wandb.Artifact(
```

Comment:
please use[ internal artifacts class ](https://github.com/wandb/wandb/blob/main/wandb/sdk/artifacts/_internal_artifact.py#L8)here instead of placeholder and switching the type